ddtrace/contrib/rq/__init__.py

```diff
@@ -0,0 +1,245 @@
+"""
+The RQ integration will trace your jobs.
+
+Usage
+~~~~~
+
+The rq integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :ref:`patch_all()<patch_all>`.
+
+Or use :ref:`patch()<patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(rq=True)
+
+
+Worker Usage
+~~~~~~~~~~~~
+
+``ddtrace-run`` can be used to easily trace your workers::
+
+    DD_RQ_WORKER_SERVICE=myworker ddtrace-run rq worker
+
+
+See https://github.com/DataDog/trace-examples/tree/kyle-verhoog/rq/python/rq
+for a working example.
+
+
+Instance Configuration
+~~~~~~~~~~~~~~~~~~~~~~
+
+
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.rq['distributed_tracing_enabled']
+
+   If ``True`` the integration will connect the traces sent between the queuer
```

Comment:
```suggestion
   If ``True`` the integration will connect the traces sent between the producer
```

? is `queuer` a term used by RQ?

Reply:
The RQ docs refer to "queues" and "workers". The app does the "enqueueing". So we could refer to the "enqueuer" and the "worker".